modules/heat_conduction/src/meshgenerators/PatchSidesetGenerator.C

```diff
@@ -133,7 +129,7 @@ PatchSidesetGenerator::generate()
 
           // this adds this node to the boundary mesh and puts it at the right position
           // in the boundary_nodes array
-          Point pt(node->operator()(0), node->operator()(1), node->operator()(2));
+          Point pt(*node);
```

Comment:
@snschune - I was able to simplify this line.

Reply:
👍